object.go

```diff
@@ -232,6 +232,7 @@ func (obj *Object) GetInt64(attr string) (int64, error) {
 }
 
 // MustGetInt64 is like GetInt64, but it panic in case of error.
+// Deprecated: use GetInt64 instead
```

Comment:
I wouldn't deprecate these functions. Similar to regexp.MustCompile for example.
https://www.journal.petrausch.info/post/2020/05/must-pattern/

Reply:
Didn't know this pattern, I like it in any case.

So what do you think about leave as is of Must* functions, deprecating URL function (it is not clear that can panic) and create a new MustURL function (or similar) to handle scenarios like

```golang
		if _, err := s.cli.GetData(MustURL("monitor/items/upload_url"), &u); err != nil {
```